src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -94,7 +93,17 @@ public void BindingWorkflow_ArgChecks()
             // Arrange
             const string QualityProfileName = "SQQualityProfileName";
             const string ProjectName = "SQProjectName";
-            BindingWorkflow testSubject = this.CreateTestSubject("key", ProjectName);
+
+            // Record all of the calls to NuGetBindingOperation.ProcessExport
+            var actualProfiles = new List<Tuple<Language, RoslynExportProfileResponse>>();
+            Mock<INuGetBindingOperation> nuGetOpMock = new Mock<INuGetBindingOperation>();
+            nuGetOpMock.Setup(x => x.ProcessExport(It.IsAny<Language>(), It.IsAny<RoslynExportProfileResponse>()))
+                .Callback<Language, RoslynExportProfileResponse>((l, r) => actualProfiles.Add(new Tuple<Language, RoslynExportProfileResponse>(l, r)))
+                .Returns(true);
+
+            BindingWorkflow testSubject = this.CreateTestSubject("key", ProjectName, nuGetOpMock.Object);
```

Comment:
Fixed up this _BindingWorkflow_ test to use a mocked _INuGetBindingOperation_ rather than calling the real code. The test is just checking that _ProcessExport_ is called with the expected data.